src/components/Chart/Chart.tsx

```diff
@@ -24,7 +24,7 @@ import { trim } from "../../helpers";
 import CustomTooltip from "./CustomTooltip";
 import ExpandedChart from "./ExpandedChart";
 
-const formatCurrency = c => {
+const formatCurrency = (c: number) => {
```

Comment:
Nit: Can we remove this function and use the `formatCurrency` function already available under `src/helpers/index.tsx`?

Reply:
Yes